src/readline.cr

```diff
@@ -48,6 +50,20 @@ module Readline
     LibReadline.rl_point
   end
 
+  def bind_key(c : Char, f : Int,Int -> Int)
```

Comment:
Note the alias in line 3, here we're outside the `lib`, so `Int != LibReadline::Int`